src/pages/Loginpage/initialValues.ts

```diff
@@ -0,0 +1,4 @@
+export const initialValues = {
+    username:'',
+    password: ''
+};
```

Comment:
Use eslint and prettier in your project to avoid this.

Reply:
Using eslint and prettier in the project now.